src/controllers/bets.ts

```diff
@@ -112,15 +112,19 @@ async function updateBet(req: Request, res: Response) {
 async function updateBetStatus(req: Request, res: Response) {
   try {
     const { id, ...body } = matchedData(req);
-    const status = body.status;
+    const status = body.status as BetEnum;
+
+    if (!Object.values(BetEnum).includes(status)) {
+      return handleHttpError(res, 'Invalid bet status', 400);
+    }
 
     await models.bets.findByIdAndUpdate(id, { $set: { status: status } });
 
     res.send({
       message: 'Bet Status Updated'
     });
   } catch (error) {
-    handleHttpError(res, 'Cannot Update Bet Status');
+    handleHttpError(res, 'Cannot Update Bet Status', 400);
```

Comment:
In this case that we have more than one possible error, should not be error code: 500?

Reply:
good catch, this should be 500

Reply:
thinking about this, is better to return a generic error from the handler 443, and just remove this code